src/app/core/routing/routing.service.ts

```diff
@@ -93,8 +93,15 @@ export class RoutingService {
           this.isBeginDateChanged(newRouterParams) || this.isEndDateChanged(newRouterParams) || this.isIntervalChanged(newRouterParams)) {
           this.loadPodcastRanks(newRouterParams);
         }
-        if (newRouterParams.group === GROUPTYPE_GEOCOUNTRY && this.isFilterChanged(newRouterParams)) {
+        if (newRouterParams.group === GROUPTYPE_GEOCOUNTRY && newRouterParams.filter &&
```

Comment:
A bit tangential to this PR, but does it make sense to set `newRouterParams` as a class property? It seems to be passed to almost every method called in the class.

Reply:
I can see where you're coming from there. this.routerParams is the equivalent, but the newRouterParams is passed around to compare what to do with it by comparing it to the most recent this.routerParams, so it's more like a decision of what to do with the event then it's being saved on the class after the decision is made.